src/cli/keepassxc-cli.cpp

```diff
@@ -130,6 +89,8 @@ int main(int argc, char** argv)
         parser.showHelp(EXIT_FAILURE);
     }
 
+    int exitCode = command->execute(argc, argv);
```

Comment:
How about removing `argv[0]` (keepassxc) and lower `argc` by `1` here instead of doing it in every Command?